camel-amq-fakeapp/README.adoc

```diff
@@ -35,8 +35,15 @@ Failure injection may be added with a future version.
 The container image can easily be build with the fabric8 build target, which uses the S2I strategy provided you are connected to OpenShift. Once you have cloned the git repository execute the following commands:
 
  $ cd auto-perf-test/camel-amq-fakeapp
+ Change amqp.host in the following file
+ camel-amq-fakeapp/src/main/openshift/camel-amq-fakeapp-props-cm.yaml
+ camel-amq-fakeapp/src/main/resources/application.properties
+
+ Create certificate as described below in section as prerequisite.
```

Comment:
Do we need that here? The certificate is not required until deployment, is it?

Reply:
Ok good point. We can have the change in application.properties made and the jks created here. It also makes sense to have it in the build phase as it is close to the code. ConfigMap and secrets can then be created during deployment. In a real project (and we may do that later) I would use an embedded broker for the component test not to have any external dependency. git clone, mvn package should just work